pkg/operator/ceph/object/config.go

```diff
@@ -129,7 +129,6 @@ func (c *clusterConfig) setDefaultFlagsMonConfigStore(rgwName string) error {
 	configOptions["rgw_log_nonexistent_bucket"] = "true"
 	configOptions["rgw_log_object_name_utc"] = "true"
 	configOptions["rgw_enable_usage_log"] = "true"
-	configOptions["rgw_frontends"] = fmt.Sprintf("%s %s", rgwFrontend(c.clusterInfo.CephVersion), c.portString(c.clusterInfo.CephVersion))
```

Comment:
Actually, it's regression and it's coming from my patch: https://github.com/rook/rook/pull/4331. Also, this one is obvious, and I should have seen it coming, it cannot be in the mon config store because it's an early config of the daemon itself.
The daemon cannot start, read the config and realize it's listening on the wrong port.
It's not going to reload itself or whatever.

Yes, this is affecting anyone using a non-default port... :-(

This is only in master so it's not a big deal since this will be fixed when 1.2 comes out.